frame/treasury/src/lib.rs

```diff
@@ -62,6 +62,7 @@ mod benchmarking;
 #[cfg(test)]
 mod tests;
 pub mod weights;
+pub mod migration;
```

Comment:
Does this need a migration? With the storage item, `on_initialize` would just correctly set the value in the first block, right?

Reply:
```suggestion
```